src/bosh-director/lib/bosh/director/errors.rb

```diff
@@ -289,10 +289,11 @@ def self.err(error_code, response_code = BAD_REQUEST)
   # Addons
   RuntimeAmbiguousReleaseSpec = err(530000)
   RuntimeInvalidReleaseVersion = err(530001)
-  RuntimeReleaseNotListedInReleases = err(530002)
+  ReleaseNotListedInReleases = err(530002)
   RuntimeInvalidDeploymentRelease = err(530003)
-  RuntimeIncompleteFilterJobSection = err(530004)
-  RuntimeIncompleteFilterStemcellSection = err(530005)
+  IncompleteFilterJobSection = err(530004)
```

Comment:
We're removing the `Runtime` namespace which makes sense, but perhaps we could keep an `Addon` prefix?